src/layout/MultipleSelect/index.tsx

```diff
@@ -1,11 +1,35 @@
 import React from 'react';
 
-import { LayoutComponent } from 'src/layout/LayoutComponent';
+import { useAppSelector } from 'src/common/hooks/useAppSelector';
+import { MultipleChoiceSummary } from 'src/layout/Checkboxes/MultipleChoiceSummary';
+import { FormComponent } from 'src/layout/LayoutComponent';
 import { MultipleSelectComponent } from 'src/layout/MultipleSelect/MultipleSelectComponent';
+import { useCommaSeparatedOptionsToText } from 'src/utils/formComponentUtils';
 import type { PropsFromGenericComponent } from 'src/layout';
+import type { SummaryRendererProps } from 'src/layout/LayoutComponent';
+import type { LayoutNodeFromType } from 'src/utils/layout/hierarchy.types';
 
-export class MultipleSelect extends LayoutComponent<'MultipleSelect'> {
+export class MultipleSelect extends FormComponent<'MultipleSelect'> {
   render(props: PropsFromGenericComponent<'MultipleSelect'>): JSX.Element | null {
     return <MultipleSelectComponent {...props} />;
   }
+
+  private useSummaryData(node: LayoutNodeFromType<'MultipleSelect'>): { [key: string]: string } {
+    const formData = useAppSelector((state) => state.formData.formData);
+    if (!node.item.dataModelBindings?.simpleBinding) {
+      return {};
+    }
+
+    const value = formData[node.item.dataModelBindings.simpleBinding] || '';
+    return useCommaSeparatedOptionsToText(node.item, value);
+  }
+
+  useDisplayData(node: LayoutNodeFromType<'MultipleSelect'>): string {
+    return Object.values(this.useSummaryData(node)).join(', ');
```

Comment:
I see we are using `Object.values(some-input).join(', ');` in many places within our codebase. Have we considered to create StringUtils.ts to make our codebase a little more readable by using descriptive utils functions instead? :D

What about creating a util-method and using it like this? 🤔
`return toCommaSeperatedString(this.useSummaryData(node)); `